src/coreclr/src/jit/importer.cpp

```diff
@@ -17494,7 +17494,7 @@ void Compiler::impImportBlockPending(BasicBlock* block)
 #ifdef DEBUG
             char buffer[400];
             sprintf_s(buffer, sizeof(buffer),
-                      "Block at offset %4.4x to %4.4x in %s entered with different stack depths.\n"
+                      "Block at offset %4.4x to %4.4x in %0.200s entered with different stack depths.\n"
```

Comment:
A string length of 200 allocated to the full method name will still leave some spare buffer capacity, even if somehow the code offset and stack depths exceed expected values.